packages/core/src/util/jwt.ts

```diff
@@ -32,13 +32,22 @@ const logger = createLogger({
  * @returns Decoded payload.
  */
 export function decodeJwt(token: string): DecodedJWT {
-  const decodedToken = jwt.decode(token);
+  return decodeJwtWithHeader(token).payload;
+}
+
+/**
+ * Decode JWT with Header.
+ * @param token - JWT to be decoded
+ * @returns Decoded payload.
+ */
+export function decodeJwtWithHeader(token: string): DecodedJWTWithHeader {
```

Comment:
Nice 👍 While we're at it, I think we might as well offer a function that just returns the whole JWT, then we'll never have to think about this again.

Reply:
I renamed the method to `decodeJwtComplete` which now also return the `signature` property which was the only one not present.